.travis.yml

```diff
@@ -14,3 +16,4 @@ script:
   - make verify
   - make smb-windows  
   - make container
+  - sudo -E env "PATH=$PATH" make integration-test
```

Comment:
looks like it's duplicated with Makefile, I will merge this PR first, pls refine it later.